tests/testNWBFile.cpp

```diff
@@ -48,5 +52,32 @@ TEST_CASE("createElectricalSeries", "[nwb]")
 
   nwbfile.finalize();
 
-  REQUIRE(result == Status::Success);
+  REQUIRE(resultCreate == Status::Success);
```

Comment:
Why is `resultCreate` checked here and not above where it is being set?

Reply:
I think that was a mistake when I added additional code, I moved it up to where it is being set